mobile/assets/i18n/en-US.json

```diff
@@ -600,5 +600,7 @@
   "download_paused": "Download paused",
   "download_enqueue": "Download enqueued",
   "download_notfound": "Download not found",
-  "download_waiting_to_retry": "Waiting to retry"
+  "download_waiting_to_retry": "Waiting to retry",
+  "crop_page_delete_orginal_title": "DDelete Original?",
```

Comment:
DDelete Original ? Double D on Delete

Reply:
Sorry, my bad. Updated it in the new commit.